nunaserver/nunaserver/server.py

```diff
@@ -23,8 +25,23 @@
 app.config["MINIO_SECURE"] = settings.MINIO_SECURE
 
 limiter.init_app(app)
-CORS(app)
 
+CORS(app, resources = {
+    r"/": {
+        "origins": settings.CORS_ALLOWED_DOMAINS
+    }
+})
+
+app.secret_key = settings.CSRF_SECRET_KEY
+csrf = SeaSurf(app)
```

Comment:
Note that the addition of CSRF protection means the frontend needs to be modified to send the CSRF token with requests; otherwise, they won't go through. Either you'll need to add support for this in the frontend, or we will have to open an issue and add it internally.

Reply:
No idea how frontend programming work, so ill file a issue